src/Avalonia.Visuals/Media/KnownColors.cs

```diff
@@ -212,7 +212,7 @@ internal enum KnownColor : uint
         Teal = 0xff008080,
         Thistle = 0xffd8bfd8,
         Tomato = 0xffff6347,
-        Transparent = 0x00ffffff,
+        Transparent = 0xffffffee,
```

Comment:
This should be reverted. You simply change the transparent color to a not commonly used color. That color is only transparent for the composer but when it's used elsewhere it isn't transparent.

Reply:
If I revert it then I can't really set the layered window key color to the old Transparent. 
I knew I have to change it, but I couldn't figure why the normal transparent "0x00ffffff" wouldn't work with SetLayeredWindowAttributes, so I hoped you guys could give me a direction in case this is not an option